tests/unit/test_views_dashboard.py

```
@@ -35,18 +56,67 @@ def test_stats_contains_expected_fields(self, api_client):
-        stats = api_client.get("/api/dashboard/").json()["stats"]
+        data = api_client.get("/api/dashboard/").json()
+        stats = data["stats"]
         assert stats["total_posts"] >= 1
         assert stats["comments"] >= 1
+        act = data["activity"]
+        assert act["latest_post_title"] == post.title
+        assert act["latest_comment_author"] == comment.author.username
+
+    def test_most_liked_posts_ranks_by_comment_upvotes(self, api_client, db, post, comment, user):
+        """most_liked_posts lists posts with at least one comment like, ordered by like total."""
+        other = User.objects.create_user(username="liker", email="liker@x.com", password="p")
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "password"

(S106)